### PR TITLE
add default value for parameters

### PR DESCRIPTION
**Pull request by ramon15749**
# Summary

add default values (completely random) 

Please check if the parameters type and range are correct

## Additional Information

Auxiliary information goes here.

## Test Plan

- Add information on how you tested your changes.

### CODE REVIEW

File: internal/clients/team3/clientconfig.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package team3
+
+func getislandParams() islandParams {
+	return islandParams{
+		giftingThreshold:            45,  // real number
```
Comment (agrimmanchanda) [inline internal/clients/team3/clientconfig.go:5]: looks fine for the moment, likely to tune post-implementation

Comment (agrimmanchanda) [inline internal/clients/team3/clientconfig.go:5]: looks fine for now, will probably change during/post implementation of function

File: internal/clients/team3/clientconfig.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		equity:                      0.1, // 0-1
+		complianceLevel:             0.1, // 0-1
+		resourcesSkew:               1.3, // >1
+		saveCriticalIsland:          true,
+		escapeCritcaIsland:          true,
+		selfishness:                 0.3, //0-1
+		minimumRequest:              30,  // real number
```
Comment (agrimmanchanda) [inline internal/clients/team3/clientconfig.go:12]: same as previous comment - this is fine for now! :)

**Review by agrimmanchanda — COMMENTED** [review]
Added comment for params team roberts will likely use.

File: internal/clients/team3/clientconfig.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+		disasterPredictionWeighting: 0.4, // 0-1
+		recidivism:                  0.3, // 0-1
```
Comment (kubitz) [inline internal/clients/team3/clientconfig.go]: 0.3 makes sense but the range actually goes in the negative. You can day -3 -> 5 (that's a bit arbitrary but it is a good range based on my desmos experimentation)

Comment (ramon15749) [reply internal/clients/team3/clientconfig.go]: so it takes real number with no bound ?

Comment (kubitz) [reply internal/clients/team3/clientconfig.go]: yup

File: internal/clients/team3/clientconfig.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		escapeCritcaIsland:          true,
+		selfishness:                 0.3, //0-1
+		minimumRequest:              30,  // real number
+		disasterPredictionWeighting: 0.4, // 0-1
+		recidivism:                  0.3, // real number
+		riskFactor:                  0.2, // 0-1
+		friendliness:                0.3, // 0-1
+		anger:                       0.2, // 0-1
+		aggression:                  0.4, // 0-1
+	}
```
Comment (kunjzk) [inline internal/clients/team3/clientconfig.go:19]: Victor and I have one more parameter - sensitivity (badly named but we couldn't think of anything better for now). Range is [0,1], sensible value is 0.5 for now :)

**Pull request merged**